src/v/pandaproxy/schema_registry/configuration.cc

```diff
@@ -42,6 +44,12 @@ configuration::configuration()
       "api_doc_dir",
       "API doc directory",
       {},
-      "/usr/share/redpanda/proxy-api-doc") {}
+      "/usr/share/redpanda/proxy-api-doc")
+  , client_cache_size(
+      *this,
+      "client_cache_size",
+      "Size of the internal kafka client cache. Default 10.",
```

Comment:
Avoid writing the default into the description: user interfaces/docs generation can pull the default from the metadata.